osu.Game/Localisation/MaintenanceSettingsStrings.cs

```diff
@@ -84,6 +119,56 @@ public static class MaintenanceSettingsStrings
         /// </summary>
         public static LocalisableString RestoreAllRecentlyDeletedModPresets => new TranslatableString(getKey(@"restore_all_recently_deleted_mod_presets"), @"Restore all recently deleted mod presets");
 
+        /// <summary>
+        /// "Deleted all collections!"
+        /// </summary>
+        public static LocalisableString DeletedAllCollections => new TranslatableString(getKey(@"deleted_all_collections"), @"Deleted all collections!");
+
+        /// <summary>
+        /// "Deleted all mod presets!"
+        /// </summary>
+        public static LocalisableString DeletedAllModPresets => new TranslatableString(getKey(@"deleted_all_mod_presets"), @"Deleted all mod presets!");
+
+        /// <summary>
+        /// "Restored all deleted mod presets!"
+        /// </summary>
+        public static LocalisableString RestoredAllDeletedModPresets => new TranslatableString(getKey(@"restored_all_deleted_mod_presets"), @"Restored all deleted mod presets!");
+
+        /// <summary>
+        /// "Everything?"
+        /// </summary>
+        public static LocalisableString MassDeleteConfirmation => new TranslatableString(getKey(@"mass_delete_confirmation"), @"Everything?");
+
+        /// <summary>
+        /// "All beatmap videos? This cannot be undone!"
+        /// </summary>
+        public static LocalisableString MassVideoDeleteConfirmation => new TranslatableString(getKey(@"mass_video_delete_confirmation"), @"All beatmap videos? This cannot be undone!");
+
+        /// <summary>
+        /// "Failed to automatically locate an osu!stable installation."
+        /// </summary>
+        public static LocalisableString StableDirectoryLocationHeader => new TranslatableString(getKey(@"stable_directory_location_header"), @"Failed to automatically locate an osu!stable installation.");
```

Comment:
I'm not sure about committing to "osu!stable" in localisations. It was done here temporarily and won't hold into the future. Needs thought.

Reply:
I see, if you think about it, osu!stable will lose its linkage later on, and it is unnecessary. I will delete it.

Reply:
The main issue is that I'm not sure what we should be calling it. "osu!stable" is probably not the correct term as it will be confusing to users, especially when "lazer" becomes "stable".

Needs further thought for sure.

Reply:
Maybe, but it's an option that will still exist when lazer is officially released, right?
If so, we need some kind of language to describe the older version, so let's discuss that separately.
I'll put this item back in asap.

Let's continue on this issue in the discussion: https://github.com/ppy/osu/discussions/20380